wings/1.0/migrating.md

```diff
@@ -51,6 +51,21 @@ rm -rf /srv/daemon
 apt -y remove nodejs # or: yum remove nodejs
 ```
 
+## Remove Standalone SFTP
+If you've used the [standalone SFTP server](/daemon/0.6/standalone_sftp.html) with the old daemon, we need to remove it as well, as it's no longer needed.
+You can do so using the following commands.
+
+```bash
+# stop and disable the standalone sftp
+systemctl disable --now pterosftp
+
+# delete the systemd service
+rm /etc/systemd/system/pterosftp.service
+
+# delete the sftp-server binary
+rm /srv/daemon/sftp-server
+```
```

Comment:
By this point, wouldn't the Daemon files already be removed thus the SFTP-Server being removed with it?

Reply:
Indeed.